java/sage/api/FavoriteAPI.java

```diff
@@ -1176,6 +1179,23 @@ public Object runSafely(Catbert.FastStack stack) throws Exception{
         return fav.getRelatedAirings(Wizard.getInstance().getRawAccess(Wizard.AIRING_CODE,
             Wizard.AIRINGS_BY_CT_CODE), true, false, new StringBuffer());
       }});
+    rft.put(new PredefinedJEPFunction("Favorite", "GetPotentialFavoriteAirings", new String[] { "Favorite" }, true)
+    {
+      /**
+       * Returns a list of all of the Airings in the database that match this Favorite.  If the favorite is disabled
+       * this API call will ignore that fact and return the Airings that would match this favorite if it were not
+       * disabled.
+       * @param Favorite the Favorite object
+       * @return the list of Airings in the DB that match this Favorite
+       *
```

Comment:
Can you add an @since annotation here so we know what version this landed on? Also, go ahead and increment the micro version in Version.java as well and then use that new value here.